exercises/csharp/TriviaGame/TriviaGame/Game.cs

```diff
@@ -19,17 +19,23 @@ public class Game
         LinkedList<string> sportsQuestions = new LinkedList<string>();
         LinkedList<string> rockQuestions = new LinkedList<string>();
 
+        string POP_CATEGORY = "Pop";
+        string SCIENCE_CATEGORY = "Science";
+        string SPORTS_CATEGORY = "Sports";
+
+        int totalQuestions = 50;
+
         int currentPlayer = 0;
         bool isGettingOutOfPenaltyBox;
 
         public Game()
         {
-            for (int i = 0; i < 50; i++)
+            for (int gameIndex = 0; gameIndex < totalQuestions; gameIndex++)
```

Comment:
I think questionIndex might be a more suitable name. It's more consistent with the name of your limit "totalQuestions"